src/main/java/io/github/perrymant/moneymaker/MoneyFormatter.java

```diff
@@ -0,0 +1,15 @@
+package io.github.perrymant.moneymaker;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+class MoneyFormatter {
+    private static final Locale UK = Locale.UK;
+
+    static String formatPounds(int amount) {
+        final double value = (double) amount / 100;
+        NumberFormat currencyInstance = NumberFormat.getCurrencyInstance(UK);
```

Comment:
This variable can be a `private static final` field - no need to re-create it every time `formatPounds` is called.